01_prepare_host.sh

```diff
@@ -229,7 +229,7 @@ fi
 # Pulling all the images except any local image.
 for IMAGE_VAR in $(env | grep -v "_LOCAL_IMAGE=" | grep "_IMAGE=" | grep -o "^[^=]*") ; do
   IMAGE="${!IMAGE_VAR}"
-  pull_container_image_if_missing "${IMAGE}"
+  pull_container_image "${IMAGE}"
```

Comment:
It is not worth having a separate function for this IMO. Just put the command directly and get rid of the function completely.
```suggestion
  sudo "${CONTAINER_RUNTIME}" pull "${IMAGE}"
```